### PR TITLE
Finalize the schedule

### LINKED ISSUE #2: Final version of the schedule
From the template README:

- [ ] Update `workshop/SCHEDULE.md` to point to appropriate materials for your workshop.
	- Add relative links to PDFs in `slides` to the schedule.
	- We recommend using [htmlpreview](https://github.com/htmlpreview/htmlpreview.github.com) to display rendered versions of R Notebooks that are tied to a specific release in the [`AlexsLemonade/training-modules`](https://github.com/AlexsLemonade/training-modules). You can use the `{{site.release_tag}}` convention used throughout the template.

### PR DESCRIPTION
**Pull request by jashapiro**
Added overly detailed time zone info & removed temporary schedule note. 

closes #2

### CODE REVIEW

**Review by jaclyn-taroni — APPROVED** [review]
:shipit: the details, ship them

File: workshop/SCHEDULE.md [hunk 1]
```diff
@@ -3,9 +3,10 @@ title: Virtual Workshop Schedule
 nav_title: Schedule
 ---
 
-<!--See an example from a past virtual workshop here: https://github.com/AlexsLemonade/2020-may-training/wiki/Schedule -->
 
-*Draft schedule subject to changes*
+The schedule for the June 2020 CCDL Virtual Training Workshop appears below.
+
+*Note: All times are [EDT (UTC−04:00)]((https://www.timeanddate.com/time/zones/edt)*
```
Comment (jaclyn-taroni) [inline workshop/SCHEDULE.md]:
```suggestion
*Note: All times are [EDT (UTC−04:00)](https://www.timeanddate.com/time/zones/edt)*
```

Comment (jaclyn-taroni) [reply workshop/SCHEDULE.md]: Oops, missed this at first glance.

File: workshop/SCHEDULE.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ The schedule for the June 2020 CCDL Virtual Training Workshop appears below.
...
 | 2:30        | Bulk RNA-seq III: [Exploratory Analyses](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/03-gastric_cancer_exploratory.nb.html) |  Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call)
-| 3:00        | Questions and introduction to the exercise: <br> [Exploratory Data Analysis](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}RNA-seq/03b-exploratory_data_analysis_exercise.Rmd) | Zoom: [Breakout Rooms](../virtual-setup/zoom-procedures.md#using-zoom-breakout-rooms) |
+| 3:00        | Questions and introduction to the exercise: <br> [Exploratory Data Analysis](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/03b-exploratory_data_analysis_exercise.Rmd) | Zoom: [Breakout Rooms](../virtual-setup/zoom-procedures.md#using-zoom-breakout-rooms) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:29]: 👀

**Comment by jaclyn-taroni** [comment]
👍 checked the links by building locally

**Pull request merged**